core/start-companion-core

```diff
@@ -5,7 +5,7 @@ SERVICES_PATH=$COMPANION_PATH/services
 
 SERVICES=(
     'cable_guy',"$SERVICES_PATH/cable_guy/main.py"
-    'wifi',"$SERVICES_PATH/wifi/main.py"
+    'wifi',"$SERVICES_PATH/wifi/main.py --socket wlan0"
```

Comment:
this hunk should not be necessary for companion

Reply:
Just tested and it indeed runs via wlan0 socket.
In fact, as I've commented [here](https://github.com/bluerobotics/companion-docker/pull/244#discussion_r636283094), the UDP socket is currently not working, so I will keep the argument so we preserve the current (working) behavior.